README.md

```diff
@@ -1,6 +1,10 @@
 # analytics-platform-control-panel
-Control panel contains admin functions like creating users and granting access to apps
+The logic/backend of the Control Panel app. Serves an API for creating users,
+apps, tools & managing permissions, which are achieved by calling through to
```

Comment:
If we're adding more details here you may also add a small mention to the fact that permissions are also updated in AWS IAM.